ui/app/components/app/asset-list-item/asset-list-item.js

```diff
@@ -131,27 +131,27 @@ const AssetListItem = ({
 };
 
 AssetListItem.propTypes = {
-  className: PropTypes.string,
+  'className': PropTypes.string,
   'data-testid': PropTypes.string,
-  iconClassName: PropTypes.string,
-  onClick: PropTypes.func.isRequired,
-  tokenAddress: PropTypes.string,
-  tokenSymbol: PropTypes.string,
-  tokenDecimals: PropTypes.number,
-  tokenImage: PropTypes.string,
-  warning: PropTypes.node,
-  primary: PropTypes.string,
-  secondary: PropTypes.string,
-  identiconBorder: PropTypes.bool,
+  'iconClassName': PropTypes.string,
```

Comment:
is it too late for me to hate this? 😆

Reply:
Definitively yes! 😂

Reply:
We're accepting PRs over at https://github.com/MetaMask/eslint-config